src/layout/header/index.js

```diff
@@ -2,6 +2,7 @@ import React from "react"
 import { StaticQuery, graphql, Link } from "gatsby"
 import './style.css';
 import AddSearch from "../../components/addSearch"
+import AddSearch2023 from "../../components/addSearch2023"
```

Comment:
I went with this 2023 file name while doing side by side comparison with the pre-existing `AddSearch`. Can we just use `AddSearch` now?